skops/conftest.py

```diff
@@ -16,3 +18,30 @@ def mock_import(name, *args, **kwargs):
 
     with patch("builtins.__import__", side_effect=mock_import):
         yield
+
+    import matplotlib  # noqa
```

Comment:
We don't need this line. First of all, it's pandas here, not matplotlib, second, we don't delete pandas from the import cache, so re-importing is not necessary.